client/src/tests/e2e/contribuer.spec.ts

```diff
@@ -38,6 +49,15 @@ test.describe("Basic form submission", () => {
     await contactEmail2.fill(contactEmail2Text);
     expect(await contactEmail2.inputValue()).toBe(contactEmail2Text);
 
+    // "Mise à jour" section
+
+    const lastUpdatedAt = page.locator("form [name=lastUpdatedAt]");
+    await lastUpdatedAt.fill("2000-05-05");
+    expect(await lastUpdatedAt.inputValue()).toBe(lastUpdatedAtDate);
+
+    const updateFrequency = page.locator("#updateFrequency");
```

Comment:
(Nit) Il y a un obstacle à utiliser

```suggestion
    const updateFrequency = page.locator("form [name=updateFrequency]");
```

comme dans les autres assertions ?